src/style.css

```diff
@@ -2,2 +1,66 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.logo,
+.nav {
+  display: block;
+}
+
+.nav {
+  align-content: center;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
```

Comment:
The font-size is set to 12px, which might be too small for readability on some devices. Consider using a relative unit like 'em' or 'rem' for better scalability.